### ISSUE TITLE
Notification is ugly !

### ISSUE THREAD

**Issue by Urgau** [linuxdeepin/developer-center#411]
Hello,

when a application show a notification but don't pass any icon, deepin will show the last icon. This is bad !

Can you solve this, please.

**Comment by hualet** [comment]
I tested it and it's working, notifications with no icon set will show an unknow application icon instead.

**Comment by hualet** [comment]
I tested it and it's working, notifications with no icon set will show an unknow application icon instead.

**Comment by Urgau** [comment]
So test with the gradio software on AUR. For having a notification you need to wait that a new sound is launch.

**Comment by hualet** [comment]
I totally don't understand. @felixonmars please help!!!

**Comment by felixonmars** [comment]
I cannot reproduce this. The icon is always the "unknown application" one even if i took a screenshot right before it.

![2017-11-21-120836-5fbf39](https://user-images.githubusercontent.com/1006477/33054551-daf4fe1a-ceb4-11e7-82dd-ce03aaed7419.png)

However occasionally the notification title was updated to a new message while the message body is still old when two messages are too close, so I got a notification showing one EURO 80's song coming from QUR'AN's radio.
